lib/fastlane/actions/slather.rb

```diff
@@ -135,7 +139,8 @@ def self.available_options
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :ignore,
                                        env_name: "FL_SLATHER_IGNORE",
-                                       description: "Tell slather to ignore files matching a path",
+                                       description: "Tell slather to ignore files matching a path or any path from an array of paths",
+                                       is_string: false,
```

Comment:
my bad! i didn't realize that was the situation! lets merge this and we can worry about the params later! :rocket: